pkg/oc/bootstrap/docker/openshift/helper.go

```diff
@@ -295,6 +298,18 @@ func (h *Helper) Start(opt *StartOptions, out io.Writer) (string, error) {
 	binds = append(binds, fmt.Sprintf("%[1]s:%[1]s", opt.DockerRoot))
 	binds = append(binds, fmt.Sprintf("%s:/var/lib/origin/openshift.local.config:z", opt.HostConfigDir))
 
+	if len(opt.OpenShiftBinaryPath) > 0 {
+		binds = append(binds, fmt.Sprintf("%s:/usr/bin/openshift:z", opt.OpenShiftBinaryPath))
+	}
+
+	// Use the host registries configuration in openshift container to allow
```

Comment:
What is this doing here?

Reply:
if you have `/etc/containers/registries.d` dir locally with registry/signature configuration and cluster up see that folder then mount it into openshift container so the signature auto-importer can read it and automatically pull the RH signatures (or other signatures..).